netlify.toml

```
@@ -6,9 +6,9 @@
   [headers.values]
     Content-Security-Policy = '''
       default-src 'self' https://*.netlify.app;
-      connect-src 'self' data: https://keybase.io https://lcd.nylira.net;
+      connect-src 'self' https://keybase.io https://lcd.nylira.net http://localhost:8010;
```

adding localhost?

---

people are going to use the proxy. It is even written in the `network.js` file as a comment

---

but not with a netlify deploy
```suggestion
      connect-src 'self' https://keybase.io https://lcd.nylira.net;
```